### PR TITLE
Add Show Compare Profile UI *

### PR DESCRIPTION
**Pull request by redls**

### CODE REVIEW

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs [hunk 1]
```diff
@@ -492,6 +492,22 @@ private static void LogUsage()
         {
             Logger.d("Received method call on DummyClient - using stub implementation.");
         }
+
+        /// <summary>
+        /// Shows the Player Games Profile UI for the given player Id.
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs]: >player Id

user identifier

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs [hunk 2]
```diff
@@ -492,6 +492,22 @@ private static void LogUsage()
...
+        /// </summary>
+        /// <param name="userId">User Identifier. This cannot be null.</param>
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs]: > This cannot be null

For consistency no need to specify this.

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/IPlayGamesClient.cs [hunk 3]
```diff
@@ -350,6 +350,15 @@ public interface IPlayGamesClient
         /// popups for achievements and other game services elements.</remarks>
         /// <param name="gravity">Gravity for the popup.</param>
         void SetGravityForPopups(Gravity gravity);
+        
+        /// <summary>
+        /// Shows the Play Games Player Profile UI for a specific player.
+        /// </summary>
+        /// </remarks>
+        /// <param name="userId">User identifier. It cannot be null.
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/BasicApi/IPlayGamesClient.cs]: Seems as param markup is broken here

File: source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/CompareProfileUiRequest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+    CompareProfileUiRequest(Player player) {
+        this.player = player;
+    }
+
+    @Override
+    protected Task<Intent> getIntent(Activity activity) {
+        GoogleSignInAccount account = HelperFragment.getAccount(activity);
+        PlayersClient playersClient = Games.getPlayersClient(activity, account);
+       return playersClient.getCompareProfileIntent(player);
```
Comment (olehkuznetsov) [inline source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/CompareProfileUiRequest.java]: > return playersClient.getCompareProfileIntent(player);

One indentation space is missing

File: source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/HelperFragment.java [hunk 5]
```diff
@@ -173,6 +174,17 @@ public static void showCaptureOverlayUi(Activity parentActivity) {
 
         return request.getTask();
     }
+    
+        public static Task<Integer> showCompareProfileUI(Activity parentActivity, Player player) {
+            CompareProfileUiRequest request = new CompareProfileUiRequest(player);
+    
+            if(!HelperFragment.startRequest(parentActivity, request)) {
```
Comment (olehkuznetsov) [inline source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/HelperFragment.java]: space after if is missing

File: source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs (hunk 1 above)
Comment (redls) [reply source/PluginDev/Assets/GooglePlayGames/BasicApi/DummyClient.cs]: changed to user identifier

**Pull request closed without merging**